src/babylon.mixins.ts

```diff
@@ -34,7 +34,7 @@ interface WebGLRenderingContext {
     vertexAttribDivisor(index: number, divisor: number): void;
 
     createVertexArray(): any;
-    bindVertexArray(vao: BABYLON.Nullable<WebGLVertexArrayObject>): void;
+    bindVertexArray(vao?: WebGLVertexArrayObject | null): void;

```

Comment:
Just curious do we really need null here as vao is anyway optional, so WebGLVertexArrayObject | undefiend;

Reply:
Good question. this was added so that I wouldn't make any changed in the engine. null is being provided as a variable in some cases, and as I didn't want to make changes that might effect the engine itself, i decided to add the null.